### ISSUE TITLE
DM: Refactor core code list ontology URIs to adhere to linked data best practices

### ISSUE THREAD

**Issue by NiclasScheiber** [IATA-Cargo/ONE-Record#268]
With the new recommended URI structure for code lists to enable querying ( #256 ), the fragments after the # of the URIs in the core code lists are too long.

Structure:
`@id: {{baseURL}}/{{codeListName}}#{{code}}`

UNCEFACT (Example):
`@id: https://vocabulary.uncefact.org/UnitMeasureCode#10`

CoreCodeLists (Unpractical):
`https://onerecord.iata.org/ns/coreCodeLists#RadioactiveMaterialClassification_SPECIAL_FORM`

To mitigate, and to not put current implementations in trouble, I suggest to deprecate the whole [coreCodeLists](https://onerecord.iata.org/ns/coreCodeLists) ontology and re-publish existing and publish all new code lists in a new Ontology, e.g. https://onerecord.iata.org/ns/code-lists

Code-Lists (New):
`https://onerecord.iata.org/ns/coreCodeLists/RadioactiveMaterialClassification#SPECIAL_FORM`

**Comment by aloccid-iata** [comment]
I am fine with the renaming but we need to have full approval

**Comment by lambertciata** [comment]
See: https://github.com/users/IATA-Cargo/projects/2/views/1?pane=issue&itemId=80230571&issue=IATA-Cargo%7CONE-Record%7C268

**Comment by yuha0lin** [comment]
So the [core code list specification](https://onerecord.iata.org/ns/coreCodeLists) is no longer used, and all code-related stuff should use [code-list](https://onerecord.iata.org/ns/coreCodeLists) specification, right?

**Comment by nscheiber-champ** [comment]
@yuha0lin this is correct. The `coreCodeLists` ontology is superseeded by `code-lists`
